language-plutus-core/examples/Language/PlutusCore/Examples/Data/InterList.hs

```diff
@@ -52,9 +51,9 @@ getBuiltinInterList = do
         . TyFun () (mkIterTyFun () [TyVar () a, TyVar () b, interlistBA] $ TyVar () r)
         $ TyVar () r
 
-getBuiltinInterNil :: Quote (Term TyName Name ())
-getBuiltinInterNil = rename =<< do
-    RecursiveType interlist wrapInterList <- getBuiltinInterList
+interNil :: Term TyName Name ()
+interNil = runQuote $ do
```

Comment:
It is safe to call `runQuote` over a type or a term that must be closed (i.e. when the type/term not being closed implies there is a bug).